### PR TITLE
Remapsies [TM ONLY FOR NOW]

### PR DESCRIPTION
**Pull request by Vizoplez**
:cl:
add: Spare reactors
balance: Lots of armory changes, LCZ has new stuff, HCZ has less stuff, EZ has some stuff changed.
add: new AIC core
/:cl:

### CODE REVIEW

**Comment by Vizoplez** [comment]
image
cable man :(

**Comment by Tsurupeta** [comment]
We have balance changelog tag for balance thingies

File: maps/site53/site53-2.dmm [hunk 1]
```diff
@@ -20581,6 +20573,12 @@
 /area/site53/ulcz/hallways)
 "bhm" = (
 /obj/structure/table/rack,
+/obj/item/ammo_magazine/scp/saiga12/stunshell,
+/obj/item/ammo_magazine/scp/saiga12/stunshell,
+/obj/item/ammo_magazine/scp/saiga12/stunshell,
+/obj/item/ammo_magazine/scp/saiga12/stunshell,
+/obj/item/ammo_magazine/box/stunshell,
+/obj/item/ammo_magazine/box/stunshell,
```
Comment (ghost) [inline maps/site53/site53-2.dmm]: based-department

Comment (Vizoplez) [reply maps/site53/site53-2.dmm]: You could already print these, but there's no harm in giving LCZ them, as they're somewhat expensive to replace AND there's only two Saigas.

**Review by hry-gh — APPROVED** [review]
looks fine but yeah i'd like some TM feedback

**Comment by checkraisefold** [comment]
better have fixed restaurant apc name and added tinted windows

**Comment by checkraisefold** [comment]
!!! FAILURE !!! [MAP: Camera unique c_tag check]: 1 issue with camera c_tags found.

**Comment by Vizoplez** [comment]
> !!! FAILURE !!! [MAP: Camera unique c_tag check]: 1 issue with camera c_tags found.

nonvital issue, also you know what causes this. Stop shitposting LOL

**Comment by checkraisefold** [comment]
> > !!! FAILURE !!! [MAP: Camera unique c_tag check]: 1 issue with camera c_tags found.
> 
> nonvital issue, also you know what causes this. Stop shitposting LOL

it makes the test fail therefore vital issue

**Pull request merged**